src/command_modules/azure-cli-iot/azure/cli/command_modules/iot/custom.py

```diff
@@ -0,0 +1,74 @@
+#---------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#---------------------------------------------------------------------------------------------
+# pylint: disable=no-self-use,too-many-arguments,no-member
+
+from __future__ import print_function
+from sys import stderr
+import azure.cli._logging as _logging
+from azure.cli._util import CLIError
+from azure.cli.command_modules.iot.mgmt_iot_hub.lib.models.iot_hub_description import \
+    IotHubDescription
+from azure.cli.command_modules.iot.mgmt_iot_hub.lib.models.iot_hub_sku_info import \
+    IotHubSkuInfo
+from azure.cli.command_modules.iot.mgmt_iot_hub_device.lib.iot_hub_device_client import \
+    IotHubDeviceClient
+from azure.cli.command_modules.iot.mgmt_iot_hub_device.lib.models.create_device_request import \
+    CreateDeviceRequest
+from azure.cli.command_modules.iot.sas_token_auth import SasTokenAuthentication
+from ._factory import resource_service_factory
+
+logger = _logging.get_az_logger(__name__)
+
+
+def _update_progress(current, total):
+    if total:
+        message = 'Percent complete: %'
+        percent_done = current * 100 / total
+        message += '{: >5.1f}'.format(percent_done)
+        print('\b' * len(message) + message, end='', file=stderr)
+        stderr.flush()
+        if current == total:
+            print('', file=stderr)
+
+# CUSTOM METHODS
+
+
+def iot_hub_create(client, name, resource_group_name, location=None, sku='F1', unit=1):
```

Comment:
>'F1' [](start = 73, length = 4)

This is probably fine but you could also reference the enum type in case the name ever changes. 

sku=IotHubSku.f1.value